trainer/model.py

```diff
@@ -41,8 +51,8 @@ def create_windows(dataset, look_back=1):
     """
     x, y = [], []
     for i in range(len(dataset) - look_back):
-        row = dataset[i:(i + look_back), 0]
-        row = np.append(row, dataset[i + look_back, 1:])  # appending CONT
+        row = dataset[i:(i + look_back), :]
+        # row = np.append(row, dataset[i + look_back, 1:])  # appending CONT
```

Comment:
@gregaw should you be using data from the day that you actually predict the sales for?